RELEASENOTES.md

```diff
@@ -6,6 +6,173 @@ This document contains the release notes for `rippled`, the reference server imp
 
 Have new ideas? Need help with setting up your node? [Please open an issue here](https://github.com/xrplf/rippled/issues/new/choose).
 
+# Version 2.2.3
```

Comment:
We will eventually need to add 2.3 section where we mention
- refactoring which affects projects with dependencies on this project
- removal of reporting mode
- removal of shards support

Not sure where we put the list of new amendments ?
featureNFTokenMintOffer
fixReducedOffersV2
fixEnforceNFTokenTrustline
fixInnerObjTemplate2
featureInvariantsV1_1
fixNFTokenPageLinks

This can wait until release candidate, so just mentioning with **no change requested**.

Reply:
Yes, I think the new amendments should be listed here under a new 2.3 section, in addition to being added on https://xrpl.org/resources/known-amendments which is the page I most frequently reference when it comes to amendments.

Those sound like breaking changes; I wonder if it's time for rippled 3.0 😀

Though now that there is Clio, I doubt anyone uses Reporting Mode anymore; I suppose 2.3 is fine.